calculator/src/App.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect } from 'react';
+
+import Container from 'react-bootstrap/Container'
+import Col from 'react-bootstrap/Col'
+import Row from 'react-bootstrap/Row'
+
+import "./app.css"
+
+function App() {
+
+  const [partyNumber, setPartyNumber] = useState(5)
+  const [fullCheckAmount, setFullCheckAmount] = useState(0)
+  const [tip, setTip] = useState(0)
+
+  const addOnePerson = () => setPartyNumber(partyNumber + 1)
+  const removeOnePerson = () => setPartyNumber(partyNumber - 1)
+
+  const handleCheckChange = e => setFullCheckAmount(e.target.value)
+  const handleTipChange = e => setTip(e.target.value)
+
+  const perPerson = parseFloat(fullCheckAmount) * ((parseFloat(tip) / 100) + 1) / partyNumber
```

Comment:
you could add a button to calculate everything where it calls a separate function and calculates perPerson amount too. You'd need to have a state for that too tho.
``` javascript
  const [perPerson, setPerPerson] = useState("");
```

Reply:
Yes... i saw it in your repo... definitely for the next one!